cmd/cluster/resizecontrolplanenode.go

```diff
@@ -253,6 +253,8 @@ func (o *resizeControlPlaneNodeOptions) forceDrainNode(nodeID string, reason str
 	// As RunElevate is unsetting KUBECONFIG, we need to redefined it if it was defined
```

Comment:
**Question:** Why do we need to unset it if `!o.kubeconfigDefined`? The comment isn't clear on that. 

nitpick: (repeating):
```suggestion
	// As RunElevate is unsetting KUBECONFIG, we need to redefine it if it was defined
```